docs/advanced-guide/union_types.md

```diff
@@ -34,17 +37,21 @@ type DateTag struct {
 ```
 
 Though we know the type of the doc while insertion, it might not be possible to know the doc type beforehand while querying a collection that stores multiple types of docs like in the case of union types. So, we need a global type that can receive the doc for any of the union types.
+
 ```go
 type GlobalTag struct {
 	BaseTag `bson:",inline"`
 	Number  *int    `bson:",omitempty"`
 	Date    *string `bson:",omitempty" mgoType:"date"`
 }
 ```
-> [!IMPORTANT]
-> Only common fields are kept as compulsory whereas other fields are marked optional.
 
-Configure schema options for the union type entities.
+:::info
```

Comment:
This will not be rendered correctly on Github. We need to support Github markdown formatting while rendering the website.

Reply:
@harsh-2711 We don't need on github assuming you are referring to readme right?